WordPress/gutenberg
# Fix DropZone class names on drop

## What?


Removes the `is-dragging-over-element` classname on drop.

## Why?


Fixes https://github.com/WordPress/gutenberg/issues/65788.

## How?


Removes the `is-dragging-over-element` on the `dragend` event as `dragleave` isn't called on drop.

## Testing Instructions





1. Insert at least two images and a cover block (easy to see in a grid of images)
2. Open the media inserter sidebar
3. Drag and drop to replace one image
4. Drag to replace a second image
5. See that only the expected dropzone overlay is shown.

### Testing Instructions for Keyboard


## Screenshots or screencast 

https://github.com/user-attachments/assets/d14d2d3b-5e24-40fe-8b2a-5bbeb4ad0a94
